src/intTest/resources/outbound_uat_data/amendment/live-1.fhir.ignore.json

```diff
@@ -29,6 +29,11 @@
       "op": "replace",
       "path": "/address/0/line/4",
       "value": "KENT"
+    },
+    {
+      "op": "replace",
+      "path": "/address/0/postalCode",
+      "value": "ZZ7 1ZZ"
```

Comment:
This postalCode was present in edifact but missing in this fhir, so I added it. Is it right, @mattd-kainos ?